packages/editor/src/browser/editor-manager.ts

```diff
@@ -90,17 +94,21 @@ export class EditorManagerImpl implements EditorManager, WidgetFactory {
         return this.activeObserver.onEditorChanged();
     }
 
-    canHandle(uri: URI, input?: EditorInput): number {
-        return 100;
+    canHandle(uri: URI, options?: EditorOpenerOptions): number {
+        return uri.scheme === 'file' ? 100 : 0;
```

Comment:
sorry, my fault. But it should actually not restrict to `file` protocol. E.g. `git` URIs are opened through this as well. 
It should handle everything for which a `DefaultResourceProvider` returns a `Resource`. Please just revert it to always returning `100`.